### DISCUSSION TITLE
toolkit: Logos for GitOps and flux

### DISCUSSION THREAD

**Discussion by bia** [fluxcd/flux2#137] [General]
Today in the FluxCD meeting, I introduced the following as a potential GitOps toolkit icon / logo.
![gitops-logo-transparent](https://user-images.githubusercontent.com/92439/89046494-4dd57a00-d34d-11ea-9d46-a4cee212c52d.png)
This brought up some questions around the identity of our set of tools, considering that 'fluxcd' is the name of the parent project.

There is already a flux logo. Should we keep it and add the other as a GitOps logo. Should everything live under a Flux logo? Should everything live under a GitOps logo? Discuss.

**Comment by stefanprodan** [comment]
Logo looks awesome thanks Bianca 🥇 

I would use this logo for the GitOps Toolkit only until we release Flux v2, after that we could replace Flux logo with this one.

**Reply by hiddeco** [reply]
I have one small suggestion @bia. I have a "dark mode" overlay on top of the GitHub UI, and I noticed due to this that parts of the logo won't be visible on dark backgrounds. Can we change the black to something that works on light and dark backgrounds (likely some shade of grey?)

**Reply by bia** [reply]
Great idea. How about like this?
![200731bluewhite](https://user-images.githubusercontent.com/92439/89048699-78750200-d350-11ea-91f1-7204cfcd68b4.png)
![200731blueblack](https://user-images.githubusercontent.com/92439/89048694-77dc6b80-d350-11ea-8164-e13543323aae.png)

**Reply by hiddeco** [reply]
Perfection :ok_hand:

**Reply by mewzherder** [reply]
@hiddeco thx for the suggestion. ;)

**Comment by dholbach** [comment]
Is the plan that we use this logo for everything GitOps Toolkit for now and once the "v2" milestones of Flux and Helm Operator have been reached we reconsider what we use for those?

**Comment by squaremo** [comment]
> 'fluxcd' is the name of the parent project.

It's the Flux project. `fluxcd` is just the name of the organisation in github (like `argoproj`, for Argo). Regrettably, people picked the org name up and started putting it in press releases etc.

**Comment by squaremo** [comment]
I like this as a techo-pictogram, because it conveys the right meanings and isn't fussy. It comes across to me more like a node in a diagram and less like a logo, and I think I get that impression because the individual elements have been in lots of diagrams I've seen lately -- or, interpreting a little, it doesn't stick out to me as identifying a _specific_ thing, rather than a general process or entity.

I know these are to some extent opposed, but -- is there a way to keep the elegance of it as a pictogram, and pick out a detail that makes it _particular_. (People mentioned that the loops looked like "G" and "O" -- not sure if that helps or hinders ..)

**Comment by Zeratoxx** [comment]
May I use the icon for a presentation?
